e2e/scaffolding/extrinsicHelpers.ts

```diff
@@ -306,7 +306,7 @@ export class ExtrinsicHelper {
   public static api: ApiRx;
   public static apiPromise: ApiPromise;
 
-  public static async initialize(providerUrl?: string | string[] | undefined) {
+  public static async initialize(providerUrl: string | string[]) {
```

Comment:
Force there to be a provider URL to prevent unintentional connections to localhost